server/src/weaverbird/backends/mongo_translator/steps/rename.py

```diff
@@ -0,0 +1,10 @@
+from typing import List
+
+from weaverbird.pipeline.steps import RenameStep
+
+
+def translate_rename(step: RenameStep) -> List:
+    return [
+        {'$addFields': {renaming[1]: f'${renaming[0]}'} for renaming in step.to_rename},
```

Comment:
I think there is a subtle mistake here: the `for` seems outside of the dict comprehension, I think it should be inside.
Check the difference in output:
```python
>>> {'$addFields': {renaming[1]: f'${renaming[0]}'} for renaming in [['a', 'b'], ['c', 'd']]}
{'$addFields': {'d': '$c'}}
>>> {'$addFields': {renaming[1]: f'${renaming[0]}' for renaming in [['a', 'b'], ['c', 'd']]}}
{'$addFields': {'b': '$a', 'd': '$c'}}
```

To be sure not to miss that, what about enriching the test case with multiple columns being renamed at once?

Reply:
Okay, I missed the fact that we could have multiple renamings.